### DISCUSSION TITLE
Pium Backend CI Config의 실행에 대해

### DISCUSSION THREAD

**Discussion by WaiNaat** [woowacourse-teams/2023-pium#46] [뜨거운 토론]
안녕하세요 친애하는 피움 팀 여러분

![image](https://github.com/woowacourse-teams/2023-pium/assets/77872742/21e41400-780a-4c07-be91-81775071ce56)

![image](https://github.com/woowacourse-teams/2023-pium/assets/77872742/92652257-7951-4236-8979-feb43637dd3b)

Pium Backend CI Config의 설정으로 인해 프론트엔드의 PR에서도 검사를 진행하는 안타까운 사건이 발생하고 있습니다ㅠㅠ

**Comment by Choi-JJunho** [comment]
해당 문제에 대해서 Front측의 변경사항에 Backend 코드가 함께 반영되어있는 경우에 대해서만 CI가 동작하는것으로 확인했습니다.
만약 해당 문제가 지속적으로 발생하거나 불편함을 가져온다면 그때 CI 동작과정에 대한 논의를 다시 해보는것은 어떨까요?

**Comment by WaiNaat** [comment]
오늘 클린 PR을 머지하면서도 같은 문제가 생겨서 범인을 찾아봤습니다.

![image](https://github.com/woowacourse-teams/2023-pium/assets/77872742/1fd19d2c-2186-477a-81f5-46ee40cccc9a)

이게 범인이었는데요.

![image](https://github.com/woowacourse-teams/2023-pium/assets/77872742/fc4064bd-a511-4099-b700-baef7fdf16b0)

이 설정으로 인해 저희가 병렬적으로 PR을 만들고 누군가의 PR이 머지되면 무조건 out of date 문제가 나오더라구요. 이걸 해결하는 과정에서 백엔드 코드가 넘어오면 무조건 CI가 동작하는 것 같습니다. 이 설정을 끄는 방향으로 논의를 해보면 좋을 것 같아요!

**Comment by Kim0914** [comment]
다시 한번 검토를 해봐야겠군요 🥲

**Comment by Choi-JJunho** [comment]
확실한 검토 감사합니다🙏
해당 설정을 없애는 방향으로 논의를 진행해봐야겠네요
